<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionLoadGame"/>
    <addaction name="actionLoadScenario"/>
    <addaction name="actionConnectToGame"/>
    <addaction name="separator"/>
    <addaction name="actionSaveGameAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionMinimap"/>
    <addaction name="separator"/>
    <addaction name="actionCityOutlines"/>
    <addaction name="actionCityOutput"/>
    <addaction name="actionMapGrid"/>
    <addaction name="actionNationalBorders"/>
    <addaction name="actionNativeTiles"/>
    <addaction name="actionCityFullBar"/>
    <addaction name="actionCityNames"/>
    <addaction name="actionCityGrowth"/>
    <addaction name="actionCityProductionLevels"/>
    <addaction name="actionCityBuyCost"/>
    <addaction name="actionCityTradeRoutes"/>
    <addaction name="separator"/>
    <addaction name="actionShowMenubar"/>
    <addaction name="actionShowToolbar"/>
   </widget>
   <widget class="QMenu" name="menuUnit">
    <property name="title">
     <string>Unit</string>
    </property>
    <addaction name="actionGotoTile"/>
    <addaction name="actionGotoNearestCity"/>
    <addaction name="actionGoAirlifttoCity"/>
    <addaction name="separator"/>
    <addaction name="actionAutoExplore"/>
    <addaction name="actionPatrol"/>
    <addaction name="actionSentry"/>
    <addaction name="actionUnsentryAllOnTile"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionUnload"/>
    <addaction name="actionUnloadAllFromTransporter"/>
    <addaction name="separator"/>
    <addaction name="actionSetHomeCity"/>
    <addaction name="actionUpgrade"/>
    <addaction name="actionConvert"/>
    <addaction name="actionDisband"/>
    <addaction name="separator"/>
    <addaction name="actionWait"/>
    <addaction name="actionDone"/>
   </widget>
   <widget class="QMenu" name="menuCombat">
    <property name="title">
     <string>Combat</string>
    </property>
    <widget class="QMenu" name="menuBuildBase">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Build Base</string>
     </property>
    </widget>
    <addaction name="actionFortifyUnit"/>
    <addaction name="actionBuildFortFortressBuoy"/>
    <addaction name="actionBuildAirstripAirbase"/>
    <addaction name="menuBuildBase"/>
    <addaction name="separator"/>
    <addaction name="actionPillage"/>
   </widget>
   <widget class="QMenu" name="menuWork">
    <property name="title">
     <string>Work</string>
    </property>
    <widget class="QMenu" name="menuBuildPath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Build Path</string>
     </property>
    </widget>
    <addaction name="actionBuildCity"/>
    <addaction name="actionAutoWorker"/>
    <addaction name="separator"/>
    <addaction name="actionBuildRoad"/>
    <addaction name="menuBuildPath"/>
    <addaction name="actionTransformToPlains"/>
    <addaction name="actionTransformToGrassland"/>
    <addaction name="separator"/>
    <addaction name="actionConnectWithRoad"/>
    <addaction name="actionConnectWithRailway"/>
    <addaction name="actionConnectWithIrrigation"/>
    <addaction name="separator"/>
    <addaction name="actionTransformTerrain"/>
    <addaction name="actionCleanPollution"/>
    <addaction name="actionCleanNuclearFallout"/>
    <addaction name="actionHelpBuildWonder"/>
    <addaction name="actionEstablishTraderoute"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHandbook"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCivilisation">
    <property name="title">
     <string>Civilisation</string>
    </property>
    <addaction name="actionUnits"/>
    <addaction name="actionPlayers"/>
    <addaction name="actionCities"/>
    <addaction name="actionEconomy"/>
    <addaction name="actionResearch"/>
    <addaction name="actionSpaceship"/>
    <addaction name="actionAchievements"/>
    <addaction name="separator"/>
    <addaction name="actionConfigureShortcuts"/>
    <addaction name="actionConfigureToolbar"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionCenterView"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionScaleFonts"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuView"/>
   <addaction name="menuCamera"/>
   <addaction name="menuUnit"/>
   <addaction name="menuCombat"/>
   <addaction name="menuWork"/>
   <addaction name="menuCivilisation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSaveGameAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save Game As...</string>
   </property>
   <property name="toolTip">
    <string>Save game as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Game...</string>
   </property>
   <property name="toolTip">
    <string>New game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoadScenario">
   <property name="text">
    <string>Load Scenario...</string>
   </property>
   <property name="toolTip">
    <string>Load scenario game</string>
   </property>
  </action>
  <action name="actionLoadGame">
   <property name="text">
    <string>Load Game...</string>
   </property>
   <property name="toolTip">
    <string>Load saved game</string>
   </property>
  </action>
  <action name="actionConnectToGame">
   <property name="text">
    <string>Connect to Game...</string>
   </property>
   <property name="toolTip">
    <string>Connect to game server</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Close application and shutdown local game server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionMinimap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Minimap</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionCityOutlines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Outlines</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCityOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Output</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="actionMapGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Map Grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionNationalBorders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>National Borders</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionNativeTiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Native Tiles</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+N</string>
   </property>
  </action>
  <action name="actionCityFullBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Full Bar</string>
   </property>
   <property name="toolTip">
    <string>City Full Bar</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionCityNames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Names</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionCityGrowth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Growth</string>
   </property>
   <property name="toolTip">
    <string>City Growth</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCityProductionLevels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Production Levels</string>
   </property>
   <property name="toolTip">
    <string>City Production Levels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCityBuyCost">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Buy Cost</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionCityTradeRoutes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>City Traderoutes</string>
   </property>
   <property name="toolTip">
    <string>City Traderoutes</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionCenterView">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Center View</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionScaleFonts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scale Fonts</string>
   </property>
  </action>
  <action name="actionGotoTile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go to Tile</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionGotoNearestCity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go to Nearest City</string>
   </property>
   <property name="shortcut">
    <string>Shift+G</string>
   </property>
  </action>
  <action name="actionGoAirlifttoCity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go / Airlift to City...</string>
   </property>
   <property name="toolTip">
    <string>Go / Airlift to City</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionAutoExplore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto Explore</string>
   </property>
   <property name="toolTip">
    <string>Auto Explore</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionPatrol">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Patrol</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionSentry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sentry</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionUnsentryAllOnTile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unsentry all on Tile</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionUnload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unload</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
  </action>
  <action name="actionUnloadAllFromTransporter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unload all from Transporter</string>
   </property>
   <property name="shortcut">
    <string>Shift+U</string>
   </property>
  </action>
  <action name="actionSetHomeCity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set Home City</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionUpgrade">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
   <property name="shortcut">
    <string>Shift+U</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisband">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disband</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionWait">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Wait</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionDone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionFortifyUnit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fortify Unit</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionBuildFortFortressBuoy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build Fort/Fortress/Buoy</string>
   </property>
   <property name="toolTip">
    <string>Build Fort/Fortress/Buoy</string>
   </property>
   <property name="shortcut">
    <string>Shift+F</string>
   </property>
  </action>
  <action name="actionBuildAirstripAirbase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build Airstrip/Airbase</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionPillage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pillage...</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="actionBuildCity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build City</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionAutoWorker">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto Worker</string>
   </property>
   <property name="toolTip">
    <string>Auto Worker</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionBuildRoad">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build Road</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionTransformToPlains">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Transform to Plains</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionTransformToGrassland">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Transform to Grassland</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionConnectWithRoad">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connect with Road</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionConnectWithRailway">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connect with Railway</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionConnectWithIrrigation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connect with Irrigation</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionTransformTerrain">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Transform Terrain</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionCleanPollution">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clean Pollution</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionCleanNuclearFallout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clean Nuclear Fallout</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionHelpBuildWonder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help Build Wonder</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionEstablishTraderoute">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Establish Traderoute</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionUnits">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Units</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionPlayers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Players</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionCities">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cities</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionEconomy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Economy</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionResearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Research</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionSpaceship">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Spaceship</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionAchievements">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Achievements</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About Freeciv-KDE</string>
   </property>
  </action>
  <action name="actionHandbook">
   <property name="icon">
    <iconset theme="system-help">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Handbook</string>
   </property>
  </action>
  <action name="actionConfigureShortcuts">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Configure Shortcuts...</string>
   </property>
  </action>
  <action name="actionConfigureToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Configure Toolbar...</string>
   </property>
   <property name="toolTip">
    <string>Configure Toolbar</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionShowMenubar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="show-menu">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Show Menubar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
